tests/app/test_draft.py

```diff
@@ -63,6 +64,8 @@ def setUp(self):
 
         self.user_internal = User('ce12b958-2a5f-44f4-a6da-861e59070a31', 'internal')
         self.user_respondent = User('0a7ad740-10d5-4ecb-b7ca-3c0384afb882', 'respondent')
+        case_service.use_mock_service()
```

Comment:
For specific unit test you may be right , a mocked response might be an easier way to go , its certainly more elegant . However we also have the situation where we want to be able to run locally without having a dependency on external services . For that it its much easier to use the mocked service .